lib/jobs/run-sku-graph.js

```diff
@@ -76,17 +75,23 @@ function runSkuGraphJobFactory(
                 return;
             }
             if (!node.sku) {
+                // It's okay if there is no SKU, it just means there is
+                // nothing to do.
                 self._done();
                 return;
             }
 
             return waterline.skus.findOne({ id: node.sku })
             .then(function(sku) {
                 if (!sku) {
+                    self._done();
```

Comment:
If we throw an error, that would mean that all discovery graphs would fail against nodes without a SKU assigned, which I think is behavior we definitely don't want. One option would be to mark the run-sku-graph task in the SKU discovery graph with `ignoreFailure: true`, but then we would ignore legitimate failure scenarios as well.